procedures/accounts_setup.rst

```diff
@@ -0,0 +1,19 @@
+Here is a list of accounts and access needed to be successful in Release Engineering
+
+General:
+ * `JIRA <https://jira.mozilla.com/projects/RELENG/issues>`_ - add username to Mozilla instance
+ * `Commit Access <https://www.mozilla.org/en-US/about/governance/policies/commit/access-policy/>`_ - fill out form, file bug
+ * Bugzilla
+ * Phabricator
+ * Github
+ * `Mozilla VPN <https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=30769829>`_
+
+Release Duty:
+ * ?
+
+Mailing Lists:
+ * 
```

Comment:
Wondering whether we should refactor this into `Communication` instead and port over content from Mail, ML, Calendar and Mail filtering from https://wiki.mozilla.org/ReleaseEngineering/Day_1_Checklist#Communication ?